package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@zowe/cics_monorepo",
   "private": true,
-  "version": "6.0.0-next.202402271249",
+  "version": "5.0.4",
```

Comment:
Hey, I do believe that we need to keep the 6.0.0-next prerelease here to avoid publishing a 5.x.x-next version from the `next` branch.

I don't mind fixing this on the fix/icons-v3 if you'd like me to

Reply:
I don't know why that changed. I'll change it back.